src/components/Marketplace/MarketplaceScreen.js

```diff
@@ -1,5 +1,104 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Airtable from 'airtable';
+import {
+  AppBar, IconButton, InputBase, Tab, Tabs, Toolbar,
+} from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import './MarketplaceScreen.css';
+import { makeStyles } from '@material-ui/core/styles';
+import FarmCard from './FarmCard';
+
+// Airtable set-up
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+// Custom styling
+const styles = makeStyles({
+  tabBackground: {
+    backgroundColor: 'grey',
+    color: 'green',
+  },
+});
 
 export default function MarketplaceScreen() {
-  return <h1>Marketplace Screen</h1>;
+  const [farmListings, setFarmListings] = useState([]);
+  // Get records from Airtable whenever DOM mounts and updates/changes
+  useEffect(() => {
+    base('Farms').select({ view: 'Grid view' }).all()
+      .then((records) => {
+        setFarmListings(records);
+      });
+  });
+
+  // 3 tab states: 0 (purchase buy), 1 (aggregate buy), 2 (distressed buy)
```

Comment:
This is no longer pertinent! You can reduce this into two tab states, as shown in the Figma which showcases Shop All and Shop by Farm!
<img width="330" alt="Screen Shot 2021-02-11 at 2 10 38 AM" src="https://user-images.githubusercontent.com/59102166/107623545-59fc9700-6c0e-11eb-92bf-f4143cae6188.png">